server/historian/packages/historian-base/package.json

```diff
@@ -17,17 +17,19 @@
     "lint": "npm run eslint",
     "lint:fix": "npm run eslint:fix",
     "start": "node dist/www.js",
-    "test": "nyc --all --temp-directory nyc/.nyc_output --cache-dir nyc/.cache --report-dir nyc/report -x dist/test/**/* mocha dist/test",
+    "test": "mocha --recursive dist/test",
```

Comment:
[Routerlicious tests scripts](https://github.com/microsoft/FluidFramework/blob/main/server/routerlicious/packages/routerlicious-base/package.json#L22) use the [make-promises-safe](https://www.npmjs.com/package/make-promises-safe) package to force an exit code 1 on unhandledPromiseRejections in tests. I think it'd be a good addition here 😃